### PR TITLE
core: add new select component

### PR DESCRIPTION
**Pull request by dschaller**
### Description

Add a select component to @clutch/core.

Screen Shot 2020-09-10 at 5 01 31 PM

Screen Shot 2020-09-10 at 5 01 27 PM

This is just a first pass at adding a basic select. In follow up PRs we can add validation and error messaging.

### CODE REVIEW

File: frontend/packages/core/src/Input/select.tsx [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+  max-width: ${props["data-max-width"] || "500px"};
+  `}
+`;
+
+interface SelectProps {
+  defaultOption?: string;
+  label: string;
+  maxWidth?: string;
+  name: string;
+  options: string[];
```
Comment (Augustyniak) [inline frontend/packages/core/src/Input/select.tsx:42]: would it make sense to be able to specify options as as list of `labels/values`? Pretty often, we want to have a dropdown that displays one string but is backed-up by some internal value that's not visible to a user. Something like:


```typescript
interface SelectProps {
    //...
    options: Option[]
    //...
}

interface Option {
    label: string
    value: string
}
```

Comment (Augustyniak) [reply frontend/packages/core/src/Input/select.tsx:42]: Obviously, this is not a blocker - just a proposal for a small modification of API.

Comment (dschaller) [reply frontend/packages/core/src/Input/select.tsx:42]: 💯

Comment (Augustyniak) [reply frontend/packages/core/src/Input/select.tsx:42]: Happy to help with the change - let me know and I can commit changes and push them.

Comment (dschaller) [reply frontend/packages/core/src/Input/select.tsx:42]: Should have it finished this morning :)

**Review by Augustyniak — APPROVED** [review]

**Review by danielhochman — APPROVED** [review]

**Pull request merged**